python/fnllm/fnllm/openai/llm/chat_text.py

```diff
@@ -152,23 +156,24 @@ async def _execute_llm(
             local_model_parameters
         )
 
-        completion = await self._call_completion_or_cache(
+        response = await self._call_completion_or_cache(
             name,
             messages=messages,
             parameters=completion_parameters,
             bypass_cache=bypass_cache,
         )
+        completion = response.value
 
-        response = completion.choices[0].message
-
-        return OpenAIChatOutput(
-            raw_input=prompt_message,
-            raw_output=response,
-            content=response.content,
-            usage=LLMUsageMetrics(
+        result = completion.choices[0].message
+        usage: LLMUsageMetrics | None = None
```

Comment:
Maybe we should still issue an `LLMUsageMetrics` even here there is a hit, but with 0 input/tokens?

Reply:
Also for the other places where this is done.